DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/Index.cshtml

```diff
@@ -18,7 +19,13 @@
 }
 
 <div class="nhsuk-grid-row">
-<div class="nhsuk-grid-column-full">
+<div class="nhsuk-grid-column-one-quarter">
+  <nav class="side-nav-menu" aria-label="Side navigation bar">
+    <partial name="~/Views/TrackingSystem/Centre/Shared/_CentreSideNavMenu.cshtml" model="CentrePage.CentreConfiguration" />
+  </nav>
+</div>
+
+<div class="nhsuk-grid-column-three-quarters">

```

Comment:
Something similar to the dashboard index file has happened here too, there are a lot of miss-aligned elements and elements that visual studio complains have no corresponding start/end tag